.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,19 @@
-Issue #, if available:
+### Issue
 
-Description of changes:
+<!-- Please link the GitHub issues related to this PR, if available -->
+
+### Description of changes
+
+<!-- Please explain the changes you made here. -->
+
+### Checklist
+- [ ] Added/modified documentation as required (such as the `README.md` for modified charts)
+- [ ] Bumped the chart `version` in `Chart.yaml` for the modified chart(s)
```

Comment:
maybe change "Bumped" to "Incremented"? Is "bumped" widely understood, especially for ESL users?

Reply:
++, will fix it. Thanks!

Reply:
Fixed: https://github.com/aws/eks-charts/pull/423